src/cmds/create_plots.py

```diff
@@ -93,28 +106,27 @@ def main():
     else:
         sk_seed = token_bytes(32)
```

Comment:
Consider parameterizing this so we can more easily create reproducible plots for tests, or for folks who want their secret keys to be child keys.

Reply:
OK